File: sub-model/src/main/java/org/github/base/mapper/ISysUserMapper.java
package org.github.base.mapper;

import org.github.base.entity.SysUserEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 用户信息表 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2019-12-12
 */
@MyBatisMapper
public interface ISysUserMapper extends IMapper<SysUserEntity> {}

File: sub-mysql-sccore-service/src/main/java/org/github/mysql/sccore/base/mapper/ISysUserMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.sccore.base.mapper;

import org.github.mysql.sccore.base.entity.SysUserEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 用户管理 Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ISysUserMapper : IMapper<SysUserEntity>
